### ISSUE TITLE
Crash after placing Garden Cloche with Transfer Node and reloading World

### ISSUE THREAD

**Issue by Watschman** [imrgr/rise-of-the-pack#5]
Pack Version: 1.3.0
Java Version: 8-144 (64bit)
Additional Mods Added: Fullscreen Windowed (Borderless) for Minecraft
Type of issue: Crash (caused by Immersive Engineering/Extra Utils 2 Compatibility)
-------------------------
Hello, 
I have updated today from Version 1.2.0 to 1.3.0 . At first I was unable to load my world due to instant crashes. So I did a lot of loading backups, updating and downgrading to get everything working at the newer version but.. I had to downgrade Immersive Engineering back to Version 72 because on Version 74 it will cause Rendering Client Crash.
I might also note that all of my Garden Cloches get Water through Extra Utilities Transfer Nodes which might be related.

Crashlog as follows 
https://pastebin.com/azqURXiA

My Solution after spending 3hours updating is to simply downgrade Immersive Engineering to 72. 

Edit: apparently it is caused by Mod - incompatibility with Immersive and Extra Utils 2. The Culprit seems to be Extra Utils 2 Transfer Nodes in combination with Immersive's Garden Cloches.

So long, 
Watschman.

**Comment by Watschman** [comment]
If you need something to replicate the issue. I have a copy of my world with Version 1.2.0. With that you should be able to replicate the crash when updating to 1.3.0. 
https://www.dropbox.com/s/t59do71brkrhrcc/Twitch%20Playthrough%20-%20Copy.zip?dl=0
(Needed to Upload it to Dropbox since Github only allows up to 10mb)

**Comment by Watschman** [comment]
Seems like the actual isssue is on Extra Utils 2 end with the Transfernode, see rwtema/extrautilities#1730 , still downgrading immersive to a previous version worked for me for now.

**Comment by imrgr** [comment]
I was able to replicate and it seems to be a bad interaction between IE and EU. I tried down grading EU to the version from the previous pack version, but that did not fix it. Something must have changed on the IE side that EU transfer nodes don't like. It seems downgrading IE is the solution until either IE or EU updates and adds a fix for it. Will confirm and upload new version of the pack shortly.
Thanks for all your help, Watschman!
